README.md

```diff
@@ -125,6 +125,7 @@ The notes are sorted alphabetically, major (upper case) notes before minor
 | AIO liquid cooler  | [NZXT Kraken Z53, Z63, Z73](docs/kraken-x3-z3-guide.md) | USB & USB HID | <sup>_p_</sup> |
 | DDR4 DRAM          | [Corsair Vengeance RGB](docs/ddr4-guide.md) | SMBus | <sup>_Uax_</sup> |
 | DDR4 DRAM          | [Generic DDR4 temperature sensor](docs/ddr4-guide.md) | SMBus | <sup>_Uax_</sup> |
+| Fan/LED controller | [Aquacomputer D5 Next](docs/aquacomputer-d5next-guide.md) | USB HID | <sup>_ehn_</sup> |
```

Comment:
It's category is more of a "watercooling pump/fan controller", since it only controls it's own LEDs... Is the existing text OK or should I write it differently?

Reply:
I think "Fan/pump controller" might be a reasonably appropriate and short name, and it can also replace the "Fan/AIO controller" category used for the NZXT H1 V2.

Reply:
Thanks, marked both as that.